gamefixes-steam/375900.py

```diff
@@ -0,0 +1 @@
+33230.py
```

Comment:
Create a separate fix for this. I believe we only want to create symbolic links within the same dir if the titles are part of the same series